external-crates/move/crates/move-compiler/src/parser/ast.rs

```diff
@@ -2079,8 +2079,11 @@ impl AstDebug for Exp_ {
                 rhs.ast_debug(w);
             }
             E::Abort(e) => {
-                w.write("abort ");
-                e.ast_debug(w);
+                w.write("abort");
+                e.as_ref().map(|e| {
```

Comment:
nit, this might be a bit more consistent for how the rest of these are written
```suggestion
                if let Some(e) = e {
```